### PR TITLE
LPS-185740

### PR DESCRIPTION
**Pull request by michaelmdelcavalcanti**

### CODE REVIEW

**Comment by michaelmdelcavalcanti** [comment]
ci:test:sf

**Comment by michaelmdelcavalcanti** [comment]
ci:test:relevant

**Comment by michaelmdelcavalcanti** [comment]
@kevhlee, could you to check my pull request?

**Comment by kevhlee** [comment]
Taking a look. @michaelmdelcavalcanti, did you still want me to take look at this pull request?

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava [hunk 1]
```diff
@@ -48,6 +54,30 @@ import com.liferay.portlet.journal.service.JournalContentSearchLocalServiceUtil;
...
+)
+public class UpgradeJavaCheck extends BaseTableClayDataSetDisplayView {
+
+	@Override
+	public ClayTableSchema getClayTableSchema() {
+		ClayTableSchemaBuilder clayTableSchemaBuilder =
+			_clayTableSchemaBuilderFactory.create();
+
+		ClayTableSchemaField nameField =
+			clayTableSchemaBuilder.addClayTableSchemaField("name", "name");
```
Comment (kevhlee) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaCheck.testjava:75]: This is the SF issue Brian Chan was talking about. It should be this:

```java
ClayTableSchemaField nameClayTableSchemaField =
        clayTableSchemaBuilder.addClayTableSchemaField("name", "name");
```

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava [hunk 2]
```diff
@@ -48,6 +54,30 @@ import com.liferay.journal.service.JournalContentSearchLocalServiceUtil;
...
+)
+public class UpgradeJavaCheck extends BaseTableFDSView {
+
+	@Override
+	public FDSTableSchema getFDSTableSchema() {
+		FDSTableSchemaBuilder fDSTableSchemaBuilder =
+			_fDSTableSchemaBuilderFactory.create();
+
+		FDSTableSchemaField nameField =
+			fDSTableSchemaBuilder.addFDSTableSchemaField("name", "name");
```
Comment (kevhlee) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava:75]: Name this to the following:

```java
FDSTableSchemaField nameFDSTableSchemaField =
        fDSTableSchemaBuilder.addFDSTableSchemaField("name", "name");
```

**Review by kevhlee — COMMENTED** [review]
Hey @michaelmdelcavalcanti,

So, in the future, we need to format our test inputs to a certain degree. At the very least, we need to fix the SF violations that are not relevant to our test case and SF rule. In this case, it was the naming of the variables and the max 80 length column rule from the previous pull request. I recommend doing the following when creating/editing test files:

1. Create or rename the test files to use the actual file extensions (in this case, rename the `UpgradeJavaCheck.testjava` file to `UpgradeJavaCheck.java`).
2. Run SF using `gradlew formatSource`.
3. Correct any adjustments made by SF to make sure we are testing the things we want to test for.

If you need assistance with this, @nicolas-sss should be able to help.

**Comment by kevhlee** [comment]
This is what the test files should look like:

## Before:

```java
/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

package com.liferay.source.formatter.dependencies.upgrade;

import com.liferay.frontend.taglib.clay.data.set.ClayDataSetDisplayView;
import com.liferay.frontend.taglib.clay.data.set.view.table.BaseTableClayDataSetDisplayView;
import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchema;
import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaBuilder;
import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaBuilderFactory;
import com.liferay.frontend.taglib.clay.data.set.view.table.ClayTableSchemaField;
import com.liferay.portal.kernel.messaging.DestinationNames;
import com.liferay.portal.kernel.messaging.Message;
import com.liferay.portal.kernel.scheduler.SchedulerEngineHelper;
import com.liferay.portal.kernel.scheduler.SchedulerEntryImpl;
import com.liferay.portal.kernel.scheduler.SchedulerException;
import com.liferay.portal.kernel.util.StringPool;
import com.liferay.portal.model.Company;
import com.liferay.portal.model.Group;
import com.liferay.portal.model.Layout;
import com.liferay.portal.model.Portlet;
import com.liferay.portal.model.Role;
import com.liferay.portal.model.User;
import com.liferay.portal.model.UserGroup;
import com.liferay.portal.service.ClassNameLocalServiceUtil;
import com.liferay.portal.service.CompanyLocalServiceUtil;
import com.liferay.portal.service.GroupLocalServiceUtil;
import com.liferay.portal.service.LayoutLocalServiceUtil;
import com.liferay.portal.service.PortletLocalServiceUtil;
import com.liferay.portal.service.RoleLocalServiceUtil;
import com.liferay.portal.service.UserGroupLocalServiceUtil;
import com.liferay.portal.service.UserLocalServiceUtil;
import com.liferay.portal.service.permission.CommonPermission;
import com.liferay.portal.service.permission.GroupPermissionUtil;
import com.liferay.portal.service.permission.LayoutPermissionUtil;
import com.liferay.portal.theme.ThemeDisplay;
import com.liferay.portal.util.CookieUtil;
import com.liferay.portal.util.PortalUtil;
import com.liferay.portlet.documentlibrary.model.DLFileEntry;
import com.liferay.portlet.documentlibrary.service.DLFileEntryLocalServiceUtil;
import com.liferay.portlet.journal.model.JournalArticle;
import com.liferay.portlet.journal.service.JournalContentSearchLocalServiceUtil;

/**
 * @author Nícolas Moura
 * @author Michael Cavalcanti
 */
@Component(
	immediate = true,
	property = {
		"clay.data.set.display.name=commercePendingOrderItems",
		"service.ranking:Integer=" + Integer.MAX_VALUE
	},
	service = ClayDataSetDisplayView.class
)
public class UpgradeJavaCheck extends BaseTableClayDataSetDisplayView {

	@Override
	public ClayTableSchema getClayTableSchema() {
		ClayTableSchemaBuilder clayTableSchemaBuilder =
			_clayTableSchemaBuilderFactory.create();

		ClayTableSchemaField nameClayTableSchemaField =
			clayTableSchemaBuilder.addClayTableSchemaField("name", "name");

		return clayTableSchemaBuilder.build();
	}

	@Reference
	private ClayTableSchemaBuilderFactory _clayTableSchemaBuilderFactory;

}
```

## After:

```java
/**
 * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
 *
 * This library is free software; you can redistribute it and/or modify it under
 * the terms of the GNU Lesser General Public License as published by the Free
 * Software Foundation; either version 2.1 of the License, or (at your option)
 * any later version.
 *
 * This library is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
 * details.
 */

package com.liferay.source.formatter.dependencies.upgrade;

import com.liferay.frontend.data.set.view.FDSView;
import com.liferay.frontend.data.set.view.table.BaseTableFDSView;
import com.liferay.frontend.data.set.view.table.FDSTableSchema;
import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilder;
import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilderFactory;
import com.liferay.frontend.data.set.view.table.FDSTableSchemaField;
import com.liferay.portal.kernel.messaging.DestinationNames;
import com.liferay.portal.kernel.messaging.Message;
import com.liferay.portal.kernel.scheduler.SchedulerEngineHelper;
import com.liferay.portal.kernel.scheduler.SchedulerEntryImpl;
import com.liferay.portal.kernel.scheduler.SchedulerException;
import com.liferay.petra.string.StringPool;
import com.liferay.portal.kernel.model.Company;
import com.liferay.portal.kernel.model.Group;
import com.liferay.portal.kernel.model.Layout;
import com.liferay.portal.kernel.model.Portlet;
import com.liferay.portal.kernel.model.Role;
import com.liferay.portal.kernel.model.User;
import com.liferay.portal.kernel.model.UserGroup;
import com.liferay.portal.kernel.service.ClassNameLocalServiceUtil;
import com.liferay.portal.kernel.service.CompanyLocalServiceUtil;
import com.liferay.portal.kernel.service.GroupLocalServiceUtil;
import com.liferay.portal.kernel.service.LayoutLocalServiceUtil;
import com.liferay.portal.kernel.service.PortletLocalServiceUtil;
import com.liferay.portal.kernel.service.RoleLocalServiceUtil;
import com.liferay.portal.kernel.service.UserGroupLocalServiceUtil;
import com.liferay.portal.kernel.service.UserLocalServiceUtil;
import com.liferay.portal.kernel.service.permission.CommonPermission;
import com.liferay.portal.kernel.service.permission.GroupPermissionUtil;
import com.liferay.portal.kernel.service.permission.LayoutPermissionUtil;
import com.liferay.portal.kernel.theme.ThemeDisplay;
import com.liferay.portal.kernel.util.CookieUtil;
import com.liferay.portal.kernel.util.PortalUtil;
import com.liferay.document.library.kernel.model.DLFileEntry;
import com.liferay.document.library.kernel.service.DLFileEntryLocalServiceUtil;
import com.liferay.journal.model.JournalArticle;
import com.liferay.journal.service.JournalContentSearchLocalServiceUtil;

/**
 * @author Nícolas Moura
 * @author Michael Cavalcanti
 */
@Component(
	immediate = true,
	property = {
		"clay.data.set.display.name=commercePendingOrderItems",
		"service.ranking:Integer=" + Integer.MAX_VALUE
	},
	service = FDSView.class
)
public class UpgradeJavaCheck extends BaseTableFDSView {

	@Override
	public FDSTableSchema getFDSTableSchema() {
		FDSTableSchemaBuilder fDSTableSchemaBuilder =
			_fDSTableSchemaBuilderFactory.create();

		FDSTableSchemaField nameFDSTableSchemaField =
			fDSTableSchemaBuilder.addFDSTableSchemaField("name", "name");

		return fDSTableSchemaBuilder.build();
	}

	@Reference
	private FDSTableSchemaBuilderFactory _fDSTableSchemaBuilderFactory;

}
```

**Comment by michaelmdelcavalcanti** [comment]
@kevhlee, Ok, I understood.
I will to replace these variables.
I opened the same PR in your repository, I will to update that.
thx

**Pull request closed without merging**